src/main/java/com/linkedin/kmf/services/ClusterTopicManipulationService.java

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright 2020 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import com.linkedin.kmf.XinfraMonitorConstants;
+import com.linkedin.kmf.services.metrics.ClusterTopicManipulationMetrics;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.common.Node;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.requests.DescribeLogDirsResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Service monitoring the creations and deletions of Kafka Cluster's Topic.
+ */
+public class ClusterTopicManipulationService implements Service {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ClusterTopicManipulationService.class);
+  private final String _configDefinedServiceName;
+  private final int _reportIntervalSecond;
```

Comment:
Nit: I highly suggest using `java.time.Duration` whenever we need a variable to represent a time duration/interval

Reply:
👍